README.md

```diff
@@ -1 +1,180 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Tribute page] <a name="about-project"></a>
+
+
+**[Tribute page]** is a project at microverse which tests your Knowledge on code review process.
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML and CSS</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[A nice heading]**
+- **[Loads very fast]**
+- **[Responsive to different devices]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+visit my github
+check tribute-page repo
+
+### Setup
+
+Clone this repository to your desired folder:
+cd Tribute page
+
+
+
+### Install
+
+Install this project with:
+
+
+  cd <Tribute-Page>
+  git clone https://github.com/sheldonwafula/my-portfolio.git
+
+
+### Usage
+
+To run the project, execute the following command:
+
+sh Tribute-Page
+
+### Run tests
+
+To run tests, run the following command:
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Sheldon Wafula**
+
+- GitHub: [@sheldonwafula](https://github.com/githubhandle)
+- Twitter: [@sheldonwafula](https://twitter.com/twitterhandle)
+- LinkedIn: [@sheldonwafula](https://linkedin.com/in/linkedinhandle)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **[Footer at the bottom]**
+- [ ] **[Logo at the top left]**
+- [ ] **[navigation bar]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project reach me through github and I will be
+very happy to connect with you.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank my long time friend Mudasir for his
+endless support from the beginning of this project upto the end.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Please add .LICENSE to your README file. Your readme appears to have no license details.